README.md

```diff
@@ -50,7 +56,7 @@ npm run start
      "component": "provider" , 
      "credentials": {
```

Comment:
Please add `"authService": "keycloak"` in the JSON snippets.

Reply:
Other than this, it should be fine

Reply:
Ok, thanks for the feedback. I've updated the readme